src/Microsoft.Azure.WebJobs.Logging/Entities/ContainerActiveEntity.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.WindowsAzure.Storage.Table;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    // Entity to track active compute Containers during a timerange.
+    // A Container is a billable compute unit, like a virtual machine. 
+    // A container may run many functions simultaneously. 
+    public class ContainerActiveEntity : TableEntity
+    {
+        const string PartitionKeyFormat = TableScheme.ContainerActivePK;
```

Comment:
We need to make sure we design the table structure from the get go to be able to handle efficient log purge strategies. We know we'll want to be able to have retention policies, as well as the ability for customers to purge logs themselves by range. We have existing issues in the issues list tracking this.

Table Storage doesn't have a good way within a single table to be able to purge a range of items by query. E.g. see https://feedback.azure.com/forums/217298-storage/suggestions/2277650-ability-to-delete-from-table-storage-by-partition. You have to query each item for deletion one by one and delete it. In high throughput scenarios, that won't scale.